tox.ini

```diff
@@ -19,4 +19,5 @@ commands =
 deps =
     mypy
 commands =
+    python3 -m pip install types-pytz
```

Comment:
Hmm, can't this be moved in `deps`, 2 lines above?

Reply:
Addressing that here: https://github.com/Doist/python-timezones/pull/18

Reply:
Now on master. Thanks for helping us picking up the litter! :)